libs/libcommon/src/libcommon/viewer_utils/asset.py

```diff
@@ -31,8 +29,8 @@ def get_and_create_dir_path(assets_directory: StrPath, url_dir_path: str) -> Pat
     return dir_path
 
 
-def get_url_dir_path(dataset: str, config: str, split: str, row_idx: int, column: str) -> str:
-    return f"{dataset}/{DATASET_SEPARATOR}/{config}/{split}/{str(row_idx)}/{column}"
+def get_url_dir_path(dataset: str, revision: str, config: str, split: str, row_idx: int, column: str) -> str:
+    return f"{dataset}/{revision}/{config}/{split}/{str(row_idx)}/{column}"
```

Comment:
do we explicitly support revisions with `/` in them, like `refs/convert/parquet`? I think we will only have `main` and commit SHAs, but I wonder if it would break something or not if we have this kind of revision (it would create more sub-directories, and we will not be able to parse the revision from the URL anymore, unless we add DATASET_SEPARATOR before and after it).

Reply:
Not sure which type of revisions we could have (so far we don't have any in the db with `/` but I think I will add the separator after and before to ensure we don't break anything.